app/jobs/split_pdf_job.rb

```diff
@@ -3,8 +3,8 @@ class SplitPDFJob < ActiveJob::Base
 
   queue_as MarkusConfigurator.markus_job_split_pdf_queue_name
 
-  def self.on_complete_js
-    "window.location.reload.bind(window.location)"
+  def self.on_complete_js(status)
+    "window.location.reload.bind(window.location)()"
```

Comment:
In general use single-quotes unless you are using string interpolation (when double-quotes are required).